apps/air-discount-scheme/api/infra/api.ts

```diff
@@ -12,6 +12,11 @@ export const serviceSetup = (services: {
         staging: 'loftbru.staging01.devland.is',
         prod: 'loftbru.island.is',
       },
+      SI_PUBLIC_IDENTITY_SERVER_ISSUER_URL: {
```

Comment:
The `SI_PUBLIC_` prefix and `getStaticEnv` are used for react forntends that need injected environment variables like service-portal and application-system-form. It wold be better to just name it `IDENTITY_SERVER_ISSUER_URL`

Reply:
@dabbeg Can it stay in .env part of infra?

Reply:
yes